fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biomes/WeightedBiomePicker.java

```diff
@@ -1,53 +1,63 @@
-/*
- * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
```

Comment:
Why?

Reply:
I don't think we do license headers in Fabric API.

Reply:
Asie told me to

Reply:
We do do license headers in Fabric API, last I checked.

Reply:
Ah yes, this will need to be restored.

Reply:
It was removed because I rewrote the class from the ground up and forgot to put it back in when I recreated the file.

Reply:
Resolved in latest merge